tensorflow/compiler/mlir/lite/transforms/prepare_composite_functions_tf.cc

```diff
@@ -281,6 +281,9 @@ void PrepareCompositeFunctionsPass::ConvertTFImplements(FuncOp func,
     if (failed(image_warping.RewriteFunc())) {
       return signalPassFailure();
     }
+  } else {
```

Comment:
We don't need to report an error when there are no matching fused functions.

Reply:
@abattery 
These checks only get executed when a function has been explicitly decorated with the "experimental_implements" keyword.
The issue I see with not throwing an error is that, because the intent is to replace on operation with another operation of equal input and output single typo in the declaration will cause TF to quietly not do anything, and leave the original op in place. As they are input and output identical, the only way to spot it would be by manual inspection of the results tflite model, i.e. by its internal structure.

Reply:
That is an intended behavior. I would not classify the extra manual inspection as an error. Emitting an warning is enough since we would not classify the valid cases as an error.

Reply:
@abattery  Ok, I changed the error to a warning, and have the pass not fail when this happens.